www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -1841,6 +1841,10 @@
           </check001>
         </Constraints>
       </pagecontent>
+      <extension type="TextField">
+        <default>html</default>
```

Comment:
although I'm doubting we need a lot of free-text options here?

Reply:
do you mean that a drop down list would be better ?